src/pysonar_scanner/configuration/cli.py

```diff
@@ -94,417 +94,423 @@ def __create_parser(cls):
             type=str,
             help="Base branch of the pull request being analyzed",
         )
-        parser.add_argument(
+        version_control_group.add_argument(
             "--sonar-newcode-reference-branch",
             "-Dsonar.newCode.referenceBranch",
             type=str,
             help="Reference branch for new code definition",
         )
-        parser.add_argument(
+        version_control_group.add_argument(
             "--sonar-scm-revision",
             "-Dsonar.scm.revision",
             type=str,
             help="Overrides the revision, for instance, the Git sha1, displayed in analysis results",
         )
-        parser.add_argument(
-            "--sonar-build-string",
-            "-Dsonar.buildString",
-            type=str,
-            help="The string passed with this property will be stored with the analysis and available in the results of api/project_analyses/search, thus allowing you to later identify a specific analysis and obtain its key for use with api/new_code_periods/set on the SPECIFIC_ANALYSIS type",
-        )
-        parser.add_argument(
-            "--sonar-source-encoding",
-            "-Dsonar.sourceEncoding",
-            type=str,
-            help="Encoding of the source files. For example, UTF-8, MacRoman, Shift_JIS",
-        )
-        parser.add_argument(
-            "--sonar-working-directory",
-            "-Dsonar.working.directory",
-            type=str,
-            help="Path to the working directory used by the Sonar scanner during a project analysis to store temporary data",
-        )
-        parser.add_argument(
-            "--sonar-scm-force-reload-all",
-            type=bool,
-            action=argparse.BooleanOptionalAction,
-            help="Set this property to true to load blame information for all files, which may significantly increase analysis duration",
-        )
-        parser.add_argument(
-            "-Dsonar.scm.forceReloadAll",
-            type=bool,
-            help="Equivalent to --sonar-scm-force-reload-all",
-        )
-        parser.add_argument(
-            "--toml-path",
-            type=str,
-            help="Path to the pyproject.toml file. If not provided, it will look in the SONAR_PROJECT_BASE_DIR",
-        )
-
-        parser.add_argument(
-            "--sonar-project-version",
-            "-Dsonar.projectVersion",
-            type=str,
-            help="Version of the project",
-        )
-
-        parser.add_argument(
-            "--sonar-project-description",
-            "-Dsonar.projectDescription",
-            type=str,
-            help="Description of the project",
-        )
 
-        parser.add_argument(
-            "--sonar-python-version",
-            "-Dsonar.python.version",
-            type=str,
-            help="Python version used for the project",
-        )
-        parser.add_argument(
+        reports_group = parser.add_argument_group("3rd party report arguments")
```

Comment:
```suggestion
        reports_group = parser.add_argument_group("3rd party reports arguments")
```

I think it needs the s